### PR TITLE
Naming convention

### PR DESCRIPTION
**Pull request by Sud1691**
added variable file.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -127,7 +129,7 @@ resource "aws_subnet" "publicSubnetB" {
   cidr_block = "10.0.2.0/24"
 
   tags = {
-    Name = "publicSubnetB"
+    Name = "var.Public_subnet"
```
Comment (geoffrey1330) [inline main.tf:132]: Name variable not properly defined

File: main.tf [hunk 2]
```diff
@@ -118,7 +120,7 @@ resource "aws_subnet" "publicSubnetA" {
   cidr_block = "10.0.1.0/24"
 
   tags = {
-    Name = "publicSubnetA"
+    Name = "var.Public_subnet"
```
Comment (geoffrey1330) [inline main.tf:123]: Name variable not properly defined

File: main.tf [hunk 3]
```diff
@@ -99,7 +101,7 @@ resource "aws_route_table" "privateRouteTableB" {
   }
 
   tags = {
-    Name = "privateRouteTableB"
+    Name = "var.privateRouteTable"
```
Comment (geoffrey1330) [inline main.tf:104]: Name variable not properly defined

File: terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,8 @@
+instance_name = "ExampleAppServerInstance"
+igw = "TrainingIGW"
+ngw = ["TrainingNGWA","TrainingNGWB"]
+publicRouteTable = "publicRouteTableA"
+privateRouteTable = ["privateRouteTableA","privateRouteTableB"]
+Public_subnet = ["publicSubnetA","publicSubnetB"]
+Private_subnet = ["privateSubnetA","privateSubnetB"]
+vpc_name = "TrainingVPC"
```
Comment (geoffrey1330) [inline terraform.tfvars:8]: Add newline at the end of file

File: main.tf (hunk 2 above)
Comment (geoffrey1330) [inline main.tf:123]: Name variable not properly defined

File: main.tf (hunk 3 above)
Comment (geoffrey1330) [inline main.tf:104]: Name variable not properly defined

File: main.tf (hunk 1 above)
Comment (geoffrey1330) [inline main.tf:132]: Name variable not properly defined

Comment (geoffrey1330) [inline main.tf:132]: Name variable not properly defined

File: main.tf (hunk 3 above)
Comment (geoffrey1330) [inline main.tf:104]: Name variable not properly defined

File: main.tf (hunk 2 above)
Comment (geoffrey1330) [inline main.tf:123]: Name variable not properly defined

File: main.tf (hunk 1 above)
Comment (Sud1691) [reply main.tf]: updated main.tf file

File: terraform.tfvars (hunk 4 above)
[the remark this answers is not in the document]
Comment (Sud1691) [reply terraform.tfvars]: added.

File: main.tf (hunk 2 above)
Comment (Sud1691) [reply main.tf]: updated main.tf

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**